libraries/ms-drivers/src/drv120_relay.c

```diff
@@ -1,24 +1,30 @@
 #include "drv120_relay.h"
 
-// Initialize pin for output
+
+// Storage for EN pin on DRV120 to open/close relay
+static GpioAddress s_relay_pin;
+
+// Initialize pin for output to open/close relay
+// Pin taken in is the EN pin on the DRV120: high = closed, low = open
+// Relay defaults closed on initialization
```

Comment:
Nit: this should go in the header file since it's useful for users of the module - you shouldn't have to look in the implementation file to see the documentation.